cmd/harmony/main.go

```diff
@@ -574,6 +575,68 @@ func setupBlacklist() (map[ethCommon.Address]struct{}, error) {
 	return addrMap, nil
 }
 
+func setupViperConfig() {
+
+	// read from environment
+	envViper := viperconfig.CreateEnvViper()
+
+	//read from config file
+	configFileViper := viperconfig.CreateConfFileViper("$GOPATH/src/github.com/harmony-one/harmony/.hmy", "nodeconfig", "json")
```

Comment:
don't use GOPATH, just use the current directory .hmy subfolder.

Reply:
yes, changed to relative path .hmy and if it miss the default value is "", so it won't trigger some errors.